docs/ref/models/querysets.txt

```diff
@@ -2371,6 +2371,39 @@ Class method that returns an instance of :class:`~django.db.models.Manager`
 with a copy of the ``QuerySet``’s methods. See
 :ref:`create-manager-with-queryset-methods` for more details.
 
+
+``explain()``
+
+.. method:: explain(verbose=False, format=None)
```

Comment:
In PostgreSQL, there's also `COST` and `BUFFERS`. I didn't check Oracle or MySQL/MariaDB/SQLite documentation, but maybe `**extra` could deal with additional backend specific features? I'm suggesting it here, as a small note in case of no changes in the documentation will address it with a note.